src/components/store/types/EditorState.ts

```diff
@@ -0,0 +1,7 @@
+import { OutputBlockData } from '../../../../types';
+
+/**
+ * Type of the state object
+ * It contains a blocks' data by block IDs as a key
+ */
+export type EditorState = Record<string, OutputBlockData>;
```

Comment:
Looks like it contains the state only of Editor Blocks, not the whole Editor state.

Maybe something like this would more useful in future?


```ts
type EditorState = {
  blocks: Record<string, OutputBlockData>,
  // other state properties
}
```